tests/event/event.cpp

```diff
@@ -444,7 +444,9 @@ static void check_get_profiling_info_return_type() {
         uint64_t>);
 }
 
-TEST_CASE("event::get_profiling_info works as expected", "[event]") {
+// FIXME: reenable when struct information descriptors are implemented
+DISABLED_FOR_TEST_CASE(hipSYCL, ComputeCPP)
```

Comment:
```suggestion
DISABLED_FOR_TEST_CASE(ComputeCPP)
```
hipSYCL seems to work fine.

Reply:
Okay. I'll update event test when push it the local branch.